apps/mull-api/src/app/entities/user.entity.ts

```diff
@@ -11,9 +11,13 @@ import {
 import { Media } from './media.entity';
 import { Event } from './event.entity';
 import { PostReaction } from './post-reaction.entity';
-import { Field, ID, Int, ObjectType } from '@nestjs/graphql';
+import { Field, ID, Int, ObjectType, registerEnumType } from '@nestjs/graphql';
```

Comment:
since we don't need `Int` anymore
```suggestion
import { Field, ID, ObjectType, registerEnumType } from '@nestjs/graphql';
```